politeiawww/user/localdb/localdb_test.go

```diff
@@ -0,0 +1,300 @@
+package localdb
```

Comment:
There are a couple instances of exceeding the 80 column limit in this file.  Please fix them.

Reply:
Done.